ext/java/nokogiri/internals/ParserContext.java

```diff
@@ -227,9 +227,9 @@ public static class Options
   }
 
   /*
-  public static class NokogiriXInlcudeEntityResolver implements org.xml.sax.EntityResolver {
+  public static class NokogiriXIncludeEntityResolver implements org.xml.sax.EntityResolver {
```

Comment:
I'm not sure what, if any, impact changing the name of this public class is so I just thought I'd note it.

Reply:
This class is commented out since f73bd4e2 (2017-02-15) so no worries about this change.